tests/empty/test-empty.cpp

```diff
@@ -94,7 +94,10 @@ TEST_CASE("Dummy test", "[empty]")
     sql << query, use(i, ind);
 
     std::vector<int> numbers(100);
+    const std::vector<int>& cref_numbers = numbers;
```

Comment:
I'm not sure if this really belongs here, wouldn't it be better to add this to the real tests, i.e. to `common-tests.h`?

Reply:
Okay. I try

Reply:
This test not run every time? Why better move to common-tests.h?
If you insist, i will fix.